Ironwood/guides/hbm/hbm.md

```diff
@@ -0,0 +1,56 @@
+# HBM Microbnechmarks on tpu7x-8
+
+This guide provides instructions for running High Bandwidth Memory (HBM) microbenchmarks on tpu7x-8 Google Kubernetes Engine (GKE) clusters. It covers creating a node pool, running the benchmarks, and viewing the expected output.
+
+## Create Node Pools
+
+Below is the command to create a TPU node-pool containing 4 ironwood chips:
```

Comment:
[nit] Wonder if we could just reference the `2x2x1` node pool creation instructions [here](https://github.com/AI-Hypercomputer/accelerator-microbenchmarks/blob/main/Ironwood/Ironwood_Microbenchmarks_readme.md#setup)?